src/main/java/edu/illinois/library/cantaloupe/processor/Java2DUtil.java

```diff
@@ -554,6 +558,13 @@ public static BufferedImage removeAlpha(final BufferedImage inImage) {
             outImage = new BufferedImage(inImage.getWidth(),
                     inImage.getHeight(), newType);
             Graphics2D g = outImage.createGraphics();
+            try {
+                String bgColor=Configuration.getInstance().getString(FLATTEN_COLOR_CONFIG_KEY);
```

Comment:
I used a different config, because it looks like BACKGROUND_COLOR_CONFIG_KEY is specifically used for rotation. I don't have a use case for a different background color for flattening alpha channels vs rotation, but I didn't know if someone else may. I'm totally good with using the existing config. Also, just put in the try/catch because if someone uses an invalid color in the configs, it could throw an exception, but that's a style thing. I'm ok without it.